firebase-config/src/main/java/com/google/firebase/remoteconfig/FirebaseRemoteConfig.java

```diff
@@ -541,6 +545,19 @@ public Task<Void> reset() {
         });
   }
 
+  /**
```

Comment:
Maybe developers have more context, but from just reading the comment, I'm not sure what "start realtime and return registration" means

Reply:
Updated to be similar to Firestore's
https://github.com/firebase/firebase-android-sdk/blob/cf4bcfd23586e39ab9e127aeaa30bc39a9a7da66/firebase-firestore/src/main/java/com/google/firebase/firestore/DocumentReference.java#L361